### ISSUE TITLE
New data added in Postgres is not getting sync to elasticsearch

### ISSUE THREAD

**Issue by DineshS31** [toluaina/pgsync#145]
PGSync version: 2.1.1

Postgres version: 12

Elasticsearch version: 7.13.4

Redis version: 5.0.7

Python version: 3.8

Problem Description: 

Hi, we tried exactly the same example of a **Book library database** which is provided in the github page and it works perfectly fine at the initial step but whenever we try to add a new data to a Postgres database it doesn't reflected in Elasticsearch and there is no error occurred at all even though there is no changes in the Elasticsearch.

Any suggestions?

Error Message (if any): NA

```


```

**Comment by toluaina** [comment]
So for the book example when you take a look at the database schema, `author` has not direct relationship to its parent node `book`.

Instead they are connected via a through table `book_author`.

The example simply shows you how to structure normalised databases.

So you would need to include the relationship between `book` and `author` using the `through_table` attribute

**Comment by toluaina** [comment]
I admit. The error could be a bit more helpful

**Comment by DineshS31** [comment]
@toluaina as you suggested we tried to include the relationship between book and author using the book_author table as well but we are getting the same error. Please refer the screenshot.
![pgsync-error](https://user-images.githubusercontent.com/37693155/128070164-2c45764c-2b26-4925-b5e9-f15593e8f91f.png)

We too tried the example **social** from this [github](https://github.com/toluaina/pgsync/tree/master/examples/social) page and facing the same error whenever we try to modify the existing data or inserting data in postgres database.

**Comment by toluaina** [comment]
You are right. I fixed this in the master branch. Can you please confirm and I will add some tests and publish a new release.

**Comment by DineshS31** [comment]
Hi @toluaina still there is an error while updating existing records. Please check the screenshot.

We got the updated code from this link **pip install git+https://github.com/toluaina/pgsync.git**

We are using the same example **social**.

![pgsync-update-error](https://user-images.githubusercontent.com/37693155/128220291-5b0e7154-048c-403a-aff0-42581a90bca5.png)

**Comment by toluaina** [comment]
Sorry about this I thought I updated this comment. 
This has been fixed in the master branch. There was an edge case with reserved database names that caused this.
@DineshS31
